examples/next-prisma/next.config.js

```diff
@@ -2,8 +2,10 @@
 module.exports = {
   experimental: {
     instrumentationHook: true,
-    serverActions: true,
+    serverComponentsExternalPackages: ["solarwinds-apm"],
```

Comment:
This is the proper way to do this instead of patching the webpack config. Server actions are stable on Next 14